Pod/Classes/GraphQL/ARGraphQLQueryMap.generated.objc

```diff
@@ -11,52 +11,52 @@ ARGraphQLQueryNameToID(NSString *name) {
       @"InboxRefetchQuery": @"20ad4063e6e46bd80f0856f3a45e0e35",
       @"InvoicePreviewRefetchQuery": @"6aaebb5a741e71949b67417a6c71890a",
-      @"LotsByFollowedArtistsQuery": @"e9b5df59deea03c09af65212cd87ac9f",
+      @"LotsByFollowedArtistsQuery": @"cae374390a17da407223c37ef8282ea0",
       @"MessagesQuery": @"045ea41b64eb6c7df5960867574aa196",
-      @"QueryRenderersArtistQuery": @"dcd9369fe3930002a7b945f18f1193f1",
-      @"QueryRenderersBidFlowQuery": @"2dbcd357c588cb69c222f7330f9ca3de",
+      @"QueryRenderersArtistQuery": @"cbd86025936f4415c4632557c23f6bc8",
+      @"QueryRenderersBidFlowQuery": @"79c707a33aa8c1d75ad950162468be2b",
       @"QueryRenderersConversationQuery": @"e497726ea96581d8a9f121a7c97823c8",
-      @"QueryRenderersForYouQuery": @"bee98558579f6cfb3966d8b70c43fbe9",
-      @"QueryRenderersGeneQuery": @"a353a267fd57bd63e008dfb46123d020",
+      @"QueryRenderersForYouQuery": @"d397189cb2dddf6bea33b136aa9dc8dc",
+      @"QueryRenderersGeneQuery": @"2d5de73d3deef13963d3d73b8b35e9f5",
       @"QueryRenderersInboxQuery": @"927e7fd3e0d34488bc507ebd9d77bfa2",
       @"QueryRenderersInquiryQuery": @"a061d9dbefece2ef49ce726b8725c34c",
       @"QueryRenderersMyProfileQuery": @"85b61ef601f4dbaf3081af4877d57680",
       @"QueryRenderersRegistrationFlowQuery": @"eae7174e3cddc451acab97c6294292ce",
-      @"QueryRenderersSaleQuery": @"d841e27359cba5fd8adb9a1677b4e6b3",
-      @"QueryRenderersWorksForYouQuery": @"ab2f1c2a40df7163fd8f3f363efdf0da",
+      @"QueryRenderersSaleQuery": @"ac8a80d55fc33b2edfee582960e1a775",
+      @"QueryRenderersWorksForYouQuery": @"19df55069e53b411c17424bc0ba46855",
       @"RegistrationCreateBidderMutation": @"5adb7100b533a36426ccc810956fd44a",
-      @"RegistrationCreateCreditCardMutation": @"b3e540bc0741b8423a1200b7cacdc55c",
-      @"SaleArtworksGridQuery": @"fe2fcd91244ec95555407491413ecb12",
-      @"SaleQuery": @"78ac3f2a4e143dd51b39a2f9fcf28aa7",
-      @"SaleRefetchQuery": @"86ae783f8c001e4d10f4e949b3a43d49",
-      @"SalesQuery": @"6459275fa4e5cee075e83e7e0506e963",
-      @"SalesRendererQuery": @"486986327c4c2b0ce00dc1170edbe6ed",
-      @"SelectMaxBidRefetchQuery": @"16019204e77d9cc45fb2961443df41e3",
+      @"RegistrationCreateCreditCardMutation": @"c050c181a19740d6db7ff99b876e0dad",
+      @"SaleArtworksGridQuery": @"59517238e770fe265224ee1deec8e590",
+      @"SaleQuery": @"3f1b806103b7227032721cb2da58b419",
+      @"SaleRefetchQuery": @"0e9383cac29e5bef7964b82987199c72",
+      @"SalesQuery": @"aa20c51bf32f3123cc6988e88c4ad698",
+      @"SalesRendererQuery": @"29ac3456b86ebcd3c20e2a6ef7896f27",
+      @"SelectMaxBidRefetchQuery": @"add71fb5f4b666877f2e0b1415b2fb23",
       @"SendConversationMessageMutation": @"fafb72e0ce2a42a7b7eb5db013f1f6a5",
       @"UpdateConversationMutation": @"376ca21dfba9abce2459940af585be24",
-      @"WorksForYouQuery": @"03402e06719b105d2d1174c8d3b0159a"
+      @"WorksForYouQuery": @"859cfe2b573b62fd12ad8880be6e34c9"
     };
```

Comment:
I'm a bit worried about this in the PR, maybe someone else forgot to update it?

Reply:
Also slightly worried, but I guess that this doesn’t need to be in sync while people are developing until they run the build command? 🤔

Reply:
(I'm a bit unsure how this initially changed)

Reply:
If you were to run these build commands on master right now, do you see the same changes? If so, then the issue is that people probably didn't rebuild the app after changing the queries.

Reply:
Ah, that makes *a lot* of sense, and we should change the expectations here maybe - as you can run emission straight from the sim. 

Maybe it's worth adding a check for this in the CI?

Reply:
Yup, this happens on master: 

<img width="466" alt="screen shot 2018-09-21 at 1 47 11 pm" src="https://user-images.githubusercontent.com/236943/45905348-e9d2e980-bda4-11e8-88a4-c94dfe0c5efa.png">